src/components/member-page/donation/table/row-detail.js

```diff
@@ -161,13 +161,13 @@ const CardTypeDictionary = {
   '5': 'AMEX',
 }
 
-const sendReceiptDictionary = {
+const SendReceiptDictionary = {
```

Comment:
Could we follow the js style guide ([notion](https://www.notion.so/twreporter/JavaScript-Style-Guide-ca71792d06714d54af5e9ffad850bf3f?pvs=4)) for the naming

Reply:
fixed